ocs_ci/deployment/acm.py

```diff
@@ -94,9 +97,27 @@ def deploy_downstream(self):
 
         login_to_acm()
         acm_obj = AcmAddClusters()
+        if self.submariner_release_type == "unreleased":
+            old_ctx = config.cur_index
+            for cluster in get_non_acm_cluster_config():
+                config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+                self.create_acm_brew_icsp()
+            config.switch_ctx(old_ctx)
         acm_obj.install_submariner_ui()
         acm_obj.submariner_validation_ui()
 
+    def create_acm_brew_icsp(self):
+        """
+        This is a prereq for downstream unreleased submariner
+
+        """
+        icsp_data = templating.load_yaml(constants.ACM_DOWNSTREAM_BREW_ICSP)
+        icsp_data_yaml = tempfile.NamedTemporaryFile(
+            mode="w+", prefix="acm_icsp", delete=False
+        )
+        templating.dump_data_to_temp_yaml(icsp_data, icsp_data_yaml.name)
+        run_cmd(f"oc create -f {icsp_data_yaml.name}", timeout=300)
```

Comment:
we need to add wait for machine config pool.
https://github.com/red-hat-storage/ocs-ci/blob/7f16a45cb770dfd8c23d09405a4ceb71752e4200/ocs_ci/utility/utils.py#L3819

Reply:
done

Reply:
done